From 481ea4ab0d476a4aa491f99c2a4e376faddc0b03 Mon Sep 17 00:00:00 2001
From: Joe McDonnell <joemcdonnell@cloudera.com>
Date: Tue, 13 Oct 2020 12:08:18 -0700
Subject: [PATCH] IMPALA-9815: Update URL for cdh-releases-rcs maven repo

We use repository.cloudera.com to get some Cloudera-patched
depdencies required by the CDP Hadoop dependencies (e.g.
log4j, logredactor, etc). The URL for repository.cloudera.com
has changed from repository.cloudera.com/content/* to
repository.cloudera.com/artifactory/*. It is possible that the
old URL will be restored. To get things working, this updates
the cdh-releases-rcs to the new URL.

It turns out that cdh-releases-rcs contains all the
artifacts that we would otherwise get from the third-party
repository, so this replaces third-party with cdh-releases-rcs.

Testing:
 - Ran build-all-options-ub1604

Change-Id: I438305565a1e6b7515408a701e9f9e31f7cfd679
Reviewed-on: http://gerrit.cloudera.org:8080/16594
Reviewed-by: Tim Armstrong <tarmstrong@cloudera.com>
Tested-by: Impala Public Jenkins <impala-public-jenkins@cloudera.com>
---
 impala-parent/pom.xml | 16 ++++------------
 1 file changed, 4 insertions(+), 12 deletions(-)

diff --git a/impala-parent/pom.xml b/impala-parent/pom.xml
index f0660719d..3baac4d7d 100644
--- a/impala-parent/pom.xml
+++ b/impala-parent/pom.xml
@@ -148,7 +148,7 @@ under the License.
     </repository>
     <repository>
       <id>cdh.rcs.releases.repo</id>
-      <url>https://repository.cloudera.com/content/groups/cdh-releases-rcs</url>
+      <url>https://repository.cloudera.com/artifactory/cdh-releases-rcs</url>
       <name>CDH Releases Repository</name>
       <snapshots>
         <enabled>false</enabled>
@@ -177,14 +177,6 @@ under the License.
         <enabled>${env.IMPALA_TOOLCHAIN_KUDU_MAVEN_REPOSITORY_ENABLED}</enabled>
       </snapshots>
     </repository>
-    <repository>
-      <id>cloudera.thirdparty.repo</id>
-      <url>https://repository.cloudera.com/content/repositories/third-party</url>
-      <name>Cloudera Third Party Repository</name>
-      <snapshots>
-        <enabled>false</enabled>
-      </snapshots>
-    </repository>
     <repository>
       <!--
       HWX Nexus is disabled. This is a tombstone to list out why:
@@ -218,9 +210,9 @@ under the License.
   </repositories>
   <pluginRepositories>
     <pluginRepository>
-      <id>cloudera.thirdparty.repo</id>
-      <url>https://repository.cloudera.com/content/repositories/third-party</url>
-      <name>Cloudera Third Party Repository</name>
+      <id>cdh.rcs.releases.repo</id>
+      <url>https://repository.cloudera.com/artifactory/cdh-releases-rcs</url>
+      <name>CDH Releases Repository</name>
       <snapshots>
         <enabled>false</enabled>
       </snapshots>

